app/components/travel/TrainDiv/TrainDiv.tsx

```
@@ -0,0 +1,135 @@
+        const trainTypePrefix = TrainType.replace(/\d+$/, ''); // Remove numbers from the end
+
+        switch (trainTypePrefix) {
+            case 'EV':
+                return <EVIcon TrainNumber={TrainNumber} />;
+            case 'EC':
+            case 'IC':
+            case 'IR':
+                return <ItalicIcon TrainNumber={TrainNumber} TrainType={trainTypePrefix} />;
+            case 'SN':
+                return <SBahnNightIcon TrainNumber={TrainNumber} />;
+            case 'RE':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)